src/content/getting-started/observe-your-clusters-and-apps/_index.md

```diff
@@ -1,15 +1,107 @@
+
+## Step 1: Create a service monitor
+
+In case of running the `hello-world` application, there is a baked-in service monitor that scrapes the metrics from the application. You can enable it by setting `serviceMonitor.enabled` to `true` in the values of the helm chart. If you are running your own application, you need to create a service monitor that scrapes the metrics from your application. This is an example of a service monitor you can use:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    application.giantswarm.io/team: team-rocket
+    app.kubernetes.io/instance: my-service
+  name: my-service
+  namespace: my-namespace
+spec:
+  endpoints:
+  - interval: 60s
+    path: /metrics
+    port: web
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: my-service
+```
+
+An important bit to notice is the `application.giantswarm.io/team` label. This label is necessary as it's required for the `Prometheus` agent to discover the target. At the same time, it's used as unique identifier for your tenant or team.
+
+Reading the manifest, the application will be scraped every 60 seconds. It will read the metrics from the `/metrics` endpoint using the [port name](https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports) as a regular service does. Also, the `app.kubernetes.io/instance` label is used to identify the application, so it should match the application label.
+
+After applying the service monitor, you can open the `Explore` view in the `Grafana` UI and start querying the metrics from your application.
+
+![Explore application metrics](explore-application-metrics.png)
+
+__Note__: The `Grafana` UI is accessible via ingress. To know the address you can check the ingress resource in the `monitoring` namespace of platform API.
+
+## Step 2: Inspect the default dashboards
+
+The platform provides a set of default dashboards that you can use to monitor the cluster components and your application. You can access them by opening the `Dashboards` view in the `Grafana` UI and explore different options in `Giant Swarm Public Dashboards` folder.
+
+Below is an example of service monitor metrics filtered to show the `hello-world` application and how many cardinality the `promhttp_metric_handler_requests_total` metric has.
+
+![Service monitor metrics](service-monitor-metrics.png)
+
+Along with service monitor metrics, you can check metrics for DNS, Ingress, Flux, and many other components. Don't hesitate to provide feedback on the dashboards or suggest new ones.
+
+## Step 3: Create your own dashboard
+
+In some occasions, you may want to create a custom dashboard to visualize the metrics of your application. Once logged into the [platform API]({{< relref "/vintage/use-the-api/management-api/_index.md" >}}) you can create a special `ConfigMap` containing the dashboard in JSON format. It will look like this:
+
+```yaml
+apiVersion: v1
```

Comment:
This duplicates the other doc right? I'm a bit skeptical because then we will need to change it in multiple places?

Reply:
I would like to move it here and, in the tutorial, add something more advanced. WDYT?

Reply:
Sure :)